canvas_modules/common-canvas/src/common-canvas/common-canvas.scss

```diff
@@ -88,15 +83,34 @@ $bottom-panel-height: 393px;
 }
 
 .bottom-panel {
+	display: flex;
+	flex-direction: column;
+	height: $bottom-panel-height;
 	position: absolute;
+	left:0;
+	right:0;
 	bottom: 0;
-	width: 100%;
-	height: $bottom-panel-height;
-	user-select: none; /* Prevent elements from being selectable */
-	background-color: $ui-01;
 	border-top: $palette-border-width solid $palette-border-color;
 }
 
+.bottom-panel-drag {
+	background: $palette-border-color;
+	cursor: row-resize;
+	flex: 0 0 2px;
+	transition: all 0.2s ease-in;
+	&:hover {
+		background: $interactive-01;
+	}
+}
+
+.bottom-panel-contents {
+	background-color: $ui-01;
+	width: 100%;
+	user-select: none;
+	height: 100%;
+
```

Comment:
Please add a min-height property of 75px.  (I'm not sure if this needs to be in this rule set or the .bottom-panel rule set so please test it out to make sure your new property works OK. It should  restricts the user from dragging the bottom panel to be less than 75px high)

Reply:
Are you setting "min-height" ? In CSS "min-height" (which means minimum height) is different to "height".  You need to set it on `.bottom-panel` for it to take effect.